profile_library/shelly/Shelly 3EM/model.json

```diff
@@ -3,16 +3,21 @@
   "measure_method": "manual",
   "measure_device": "GW Instek GPM-8310",
   "name": "Shelly 3EM",
-  "standby_power": 1.22,
-  "sensor_config": {
-    "power_sensor_naming": "{} Device Power",
-    "energy_sensor_naming": "{} Device Energy"
-  },
+  "aliases": [
+    "SHEM-3"
+  ],
+  "discovery_by": "entity",
```

Comment:
This may be omitted btw, as this is the default. But no big deal to leave it in.